lib/_http_common.js

```diff
@@ -22,11 +22,7 @@
 'use strict';
 
 const {
-  ArrayPrototypePushApply,
```

Comment:
All the ArrayPrototype operations where clearly problematic in my analysis.

Reply:
I'm surprised that all of them would be, but I guess if that's the case it's OK to remove them all and try to add them back later if V8 optimize this away.